quinn-proto/src/connection/mod.rs

```diff
@@ -1338,7 +1337,9 @@ impl Connection {
         let loss_delay = cmp::max(rtt.mul_f32(self.config.time_threshold), TIMER_GRANULARITY);
 
         // Packets sent before this time are deemed lost.
-        let lost_send_time = now - loss_delay;
+        let lost_send_time = now
+            .checked_sub(loss_delay)
+            .expect("overflow when subtracting duration from instant");
```

Comment:
Technically this is an underflow, and in any case it's unlikely enough that an `unwrap()` is fine.